specification/app/resource-manager/Microsoft.App/stable/2022-10-01/ContainerApps.json

```diff
@@ -568,11 +568,28 @@
               "$ref": "./CommonDefinitions.json#/definitions/WorkloadProfileType",
               "description": "Workload profile type to pin for container app execution."
             },
+            "appState": {
```

Comment:
Should we introduce new schema changes via preview version first and later bring it to stable.

Reply:
@jijohn14 I think the addition of a property is a minor schema change that does not have to go through the preview cycle. I think preview API versions should be used for major API changes, such as new resource types, new complex object type properties, new API paths, etc.